dcat/index.html

```diff
@@ -3279,29 +3282,6 @@ <h2>Versioning</h2>
 <p>Versioning can be applied to any of the first class citizens DCAT resources including Catalogs, Datasets, Distributions. The notion of version is very much related to the community practices,  data management policy and the workflows in place. It is up to data providers to decide when and why a new version should be released. For this reason, DCAT refrains from providing definitions or rules about when changes in a resource should turn into a new release of it. </p>
 
 <p>Versioning may be understood as involving relationships between datasets, which is supported by the <a href="#Property:resource_qualified_relation"><code>dcat:qualifiedRelation</code></a> and described in <a href="#qualified-relationship"></a>. The class <a href="#Class:Relationship"><code>dcat:Relationship</code></a> supports providing information about the relationship, and could be extended for versioning information.</p>
-  <!--backlog p class="issue" data-number="90">
-  <p class="issue" data-number="90">
-        The need to be able to describe version relationships of datasets has been identified as a requirement to be satisfied in the revision of DCAT. Also see detailed requirements in
-        <a href="https://github.com/w3c/dxwg/issues/89">Issue #89</a>,
-        <a href="https://github.com/w3c/dxwg/issues/91">Issue #91</a>,
-        <a href="https://github.com/w3c/dxwg/issues/92">Issue #92</a>,
-        <a href="https://github.com/w3c/dxwg/issues/93">Issue #93</a>,
-    </p-->
-<!--
-<aside class="note">
-<p>
-In this section it is planned to describe some patterns for describing Dataset and/or Distribution versions.
-See the wiki page on <a href="https://github.com/w3c/dxwg/wiki/Dataset-versioning">Dataset versioning</a> for more discussion.
-</p>
-</aside>
--->
-
-<aside class="ednote">
```

Comment:
I would keep this EDNOTE, removing the reference to the issue.

Reply:
Done